content/python/concepts/built-in-functions/terms/property/property.md

```diff
@@ -0,0 +1,114 @@
+---
+Title: 'property()'
+Description: 'Declares property to access class attributes.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Classes'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`property()`** function is a built-in Python function used to create properties by defining getter, setter and deleter for class attributes. It returns the object of property class that can be used as decorator or assigned to a class attribute.
```

Comment:
```suggestion
The **`property()`** function is a built-in Python function used to define methods that get, set, and delete class attributes. It can be used as a [decorator](https://www.codecademy.com/resources/docs/python/decorators) or assigned to a class attribute.
```